README.md

```diff
@@ -370,31 +370,31 @@ Integration flow:
 
 ## Credit card integration guide
 
-> A working example can be found [here](./examples/credit-card)
-> N.B. 3DS flow doesn't work with sandbox environment, but sandbox mode is suitable for testing flow with test cards without 3DS.
+> A working example can be found [here](./examples/credit-card).
+> Note: 3The 3-D Secure flow doesn't work in a sandbox environment.
```

Comment:
"3The"?

Reply:
Also, it seems sandbox features text lost (like testing with test cards).

Reply:
> Also, it seems sandbox features text lost (like testing with test cards).

two parts of the sentence have the same meaning, so we removed the second one. 3ds doesn't work in a sandbox environments = sandbox is suitable for everything else